requirements/edx/development.txt

```diff
@@ -4,6 +4,7 @@
 
 # Python libraries to install directly from github / PyPi
 
+xlwt==1.3.0
```

Comment:
1.3.0 (22 August 2017) `Officially support Python 3.6, drop support for 2.6.`
`https://xlwt.readthedocs.io/en/latest/changes.html#august-2017`

Please use version `1.2.0 (4 January 2017)`